config.json

```diff
@@ -1,3 +1,6 @@
 {
-    "csvDelimiter": "|"
+    "csvDelimiter": "|" ,
+    "api": {
+        "eventURL": "http://api.firstrecon.tech/events"
+    }
```

Comment:
#Fix Let's change `eventURL` to `event.url`. That way if we need to add new config data it can stay in a nice hierarchical format. Also, let's change `api` to `apis`, as this field will very likely have multiple different configurations (such as `teams` and `matches`) in it at some point. This will make your `apis` field have the following content:
```
"apis": {
  "event": {
    "url": "http://api.firstrecon.tech/events"
  }
}
```